packages/scandipwa/src/route/Checkout/Checkout.container.js

```diff
@@ -264,6 +301,11 @@ export class CheckoutContainer extends PureComponent {
             });
         }
 
+        if (/billing/.test(urlStep) && /success/.test(prevUrlStep)) {
+            BrowserDatabase.deleteItem(PAYMENT_TOTALS);
+            history.push(appendWithStoreCode('/cart'));
```

Comment:
Please don't use the custom strings. Use a constant here.
I believe there should already be a constant for `/cart`. If no, please create a new one.